src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -68,6 +68,15 @@ constexpr mixxx::RgbColor kTrackColorAqua(0x16C0F8);
 constexpr mixxx::RgbColor kTrackColorBlue(0x0150F8);
 constexpr mixxx::RgbColor kTrackColorPurple(0x9808F8);
 
+struct memory_cue_t {
```

Comment:
@uklotzde Are we using C-style structs in Mixxx or would a class be prefereable?

Reply:
Happy to change to a class, was following the precedent set in the Serato feature https://github.com/mixxxdj/mixxx/blob/33cb3dd2d3bb958bc62c436385cd78816f645d01/src/library/serato/seratofeature.cpp#L71

Reply:
hehe, good point. It's probably okay then ;-)